2022/01/index.js

```diff
@@ -2,13 +2,44 @@ const { promises: fs } = require('fs');
 
 async function getInput() {
   const input = (await fs.readFile(`${__dirname}/input.txt`))
-    .toString();
+    .toString()
+    .trim();
+  const inventories = input
+    .split('\n\n')
+    .map(inventory => inventory.split('\n').map(calories => parseInt(calories)));
+
+  return inventories;
+}
+
+function sumArray(arr) {
+  return arr.reduce(
+    (acc, element) => acc + element,
+    0
+  );
```

Comment:
Does the 0 mean your accumulator starts at 0?
Also, why is it split onto 3 lines?

Reply:
Yes. One line per argument.